docs/ops/arithmetic/Round_5.md

```diff
@@ -0,0 +1,52 @@
+## Round <a name="Round"></a> {#openvino_docs_ops_arithmetic_Round_5}
```

Comment:
Please, create `docs/ops/opset5.md` document and add `Round-5` to it.

Reply:
Done.